sources/Foundation/Types/Types.h

```diff
@@ -167,6 +167,8 @@ struct FourCC {
     ActionRandomName = 100,
     ActionBootSelect = 18,
     ActionEdit = 59,
+    ActionRenderMixdown = 200,
```

Comment:
@democloid actually just realised making these 200 and 201 is not a good idea as it makes the mess with numbering even worse, will change them to be next consecutive available numbers.